### PR TITLE
Fix/eslint

### PR DESCRIPTION
**Pull request by hyperxtend**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+        },
+        'ecmaVersion': 2018,
+        'sourceType': 'module'
+    },
+    'plugins': ['react'],
+    settings: {
+        react: {
+            version: 'detect'
+        },
+        createClass: 'createReactClass',
```
Comment (strange-developer) [inline .eslintrc.js]: Do you need this here?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+        pragma: 'React',
+        flowVersion: '0.53',
```
Comment (strange-developer) [inline .eslintrc.js]: Is this required?

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+        'import/resolver': {
+            node: {
+                moduleDirectory: ['node_modules', 'src'],
+            },
+        },
+    },
+    'rules': {
+        'accessor-pairs': 'error',
```
Comment (strange-developer) [inline .eslintrc.js]: All of these rules shouldn't need to be enabled. We can turn off specific rules. It should extend airbnb's guide

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "cSpell.words": [
```
Comment (strange-developer) [inline .vscode/settings.json:2]: Add this folder into your `.gitignore`

File: src/App.test.jsx [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
+import './setup-tests';
```
Comment (strange-developer) [inline src/App.test.jsx]: You shouldn't need to import this file in every test file. Jest should use this configuration by default.

**Review by strange-developer — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (hyperxtend) [reply .eslintrc.js]: Thought I did but turns out I did not

File: .eslintrc.js (hunk 2 above)
Comment (hyperxtend) [reply .eslintrc.js]: Nope, it's not

File: .vscode/settings.json (hunk 4 above)
Comment (hyperxtend) [reply .vscode/settings.json:2]: Done that

File: src/App.test.jsx (hunk 5 above)
Comment (hyperxtend) [reply src/App.test.jsx]: Removed these imports from the test files

**Pull request merged**